### PR TITLE
i18n(es): Translate `sidebar.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Translated `sidebar.mdx` to spanish

### CODE REVIEW

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+---
+
+import FileTree from '../../../../components/file-tree.astro';
+import SidebarPreview from '../../../../components/sidebar-preview.astro';
+
+Una barra lateral bien organizada es clave para una buena documentación, ya que es una de las principales formas en que los usuarios navegarán por su sitio. Starlight proporciona un conjunto completo de opciones para personalizar el diseño y el contenido de tu barra lateral.
+
+## Barra lateral predeterminada
+
+Por defecto, Starlight generará automáticamente una barra lateral basada en la estructura del sistema de archivos de su documentación, utilizando la propiedad `title` de cada archivo como entrada de la barra lateral.
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/sidebar.mdx:15]:
```suggestion
Por defecto, Starlight generará automáticamente una barra lateral basada en la estructura del sistema de archivos de tu documentación, utilizando la propiedad `title` de cada archivo como entrada de la barra lateral.
```

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+	]}
+/>
+
+Aprende más sobre las barras laterales generadas automáticamente en la sección [grupos autogenerados](#grupos-autogenerados).
+
+## Agregar enlaces y grupos de enlaces
+
+Para configurar los enlaces de tu barra lateral [enlaces](#enlaces) y [grupos de enlaces](#grupos) (dentro de un encabezado plegable), usa la propiedad [`starlight.sidebar`](/es/reference/configuration/#sidebar) en `astro.config.mjs`.
+
+By combining links and groups, you can create a wide variety of sidebar layouts.
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/sidebar.mdx:61]: falta traducir

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+			],
+		},
+	]}
+/>
+
+Aprende más sobre las barras laterales generadas automáticamente en la sección [grupos autogenerados](#grupos-autogenerados).
+
+## Agregar enlaces y grupos de enlaces
+
+Para configurar los enlaces de tu barra lateral [enlaces](#enlaces) y [grupos de enlaces](#grupos) (dentro de un encabezado plegable), usa la propiedad [`starlight.sidebar`](/es/reference/configuration/#sidebar) en `astro.config.mjs`.
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/sidebar.mdx:59]:
```suggestion
Para configurar los [enlaces](#enlaces) de tu barra lateral y [grupos de enlaces](#grupos) (dentro de un encabezado plegable), usa la propiedad [`starlight.sidebar`](/es/reference/configuration/#sidebar) en `astro.config.mjs`.
```

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 4]
```diff
@@ -0,0 +1,487 @@
...
+By combining links and groups, you can create a wide variety of sidebar layouts.
+
+### Enlaces
+
+Agrega un enlace a una página interna o externa usando un objeto con las propiedades `label` y `link`.
+
+```js
+starlight({
+	sidebar: [
+		// Un enlace a la guía CSS & Styling.
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/sidebar.mdx:70]:
```suggestion
		// Un enlace a la guía CSS y Estilos.
```

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 5]
```diff
@@ -0,0 +1,487 @@
...
+	sidebar: [
+		// Un grupo de enlaces etiquetados como "Guides"
+		{
+			label: 'Guías',
+			items: [
+				{ label: 'Componentes', link: '/guides/components/' },
+				{ label: 'Internacionalización (i18n)', link: '/guides/i18n/' },
+				// Un grupo anidado de enlaces.
+				{
+					label: 'Styling',
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/sidebar.mdx:107]:
```suggestion
					label: 'Estilando',
```

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 6]
```diff
@@ -0,0 +1,487 @@
...
+    - docs/
+      - guides/
+        - components.md
+        - i18n.md
+        - advanced/
+          - project-structure.md
+
+</FileTree>
+
+The following sidebar will be generated:
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/sidebar.mdx:177]: falta traducir

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 7]
```diff
@@ -0,0 +1,487 @@
...
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Guías',
+			items: [
+				{ label: 'Componentes', link: '/guides/components/' },
+				{ label: 'Internacionalización (i18n)', link: '/guides/i18n/' },
+				{
+					label: 'advanced',
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/sidebar.mdx:187]:
```suggestion
					label: 'avanzado',
```

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 8]
```diff
@@ -0,0 +1,487 @@
...
+			],
+		},
+	]}
+/>
+
+#### Personalización de enlaces autogenerados en el frontmatter
+
+Usa el campo [`sidebar`](/es/reference/frontmatter/#sidebar) en las páginas individuales para personalizar los enlaces autogenerados.
+
+Las opciones del frontmatter de la barra lateral te permiten establecer un [etiqueta personalizada](/es/reference/frontmatter/#label) o agregar una [insignia](/es/reference/frontmatter/#badge), [esconder](/es/reference/frontmatter/#hidden) un enlace de la barra lateral o definir un [peso de ordenación personalizado](/es/reference/frontmatter/#order).
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/sidebar.mdx:204]:
```suggestion
Las opciones del frontmatter de la barra lateral te permiten establecer un [etiqueta personalizada](/es/reference/frontmatter/#label) o agregar una [insignia](/es/reference/frontmatter/#badge) a un enlace, [ocultar](/es/reference/frontmatter/#hidden) un enlace de la barra lateral o definir un [peso de ordenación personalizado](/es/reference/frontmatter/#order).
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 9]
```diff
@@ -0,0 +1,487 @@
...
+    - docs/
+      - guides/
+        - components.md
+        - i18n.md
+        - advanced/
+          - project-structure.md
+
+</FileTree>
+
+The following sidebar will be generated:
```
Comment (Waxer59) [inline docs/src/content/docs/es/guides/sidebar.mdx:177]: falto traducir esta linea

**Comment by dreyfus92** [comment]
Listo @Waxer59 😁

**Review by delucis — APPROVED** [review]
Thanks team! Love seeing this new guide translated already 🚀

**Pull request merged**